arduino-ide-extension/src/browser/style/main.css

```diff
@@ -15,15 +15,24 @@
     background: var(--theia-arduino-toolbar-background);
 }
 
-.p-TabBar-toolbar .item.arduino-tool-item > div:hover {
-    background: (--theia-arduino-toolbar-hoverBackground);
+.p-TabBar-toolbar .item.arduino-tool-item:hover > div {
+    background: white;
```

Comment:
We cannot use hard-coded color so neither `white` nor `rgb(255,204,0)` is acceptable. We use VS Code themes, any new colors we want to use have to be derived from the available theme colors.
 - https://code.visualstudio.com/docs/getstarted/themes
 - https://code.visualstudio.com/api/extension-capabilities/theming
 - https://code.visualstudio.com/api/references/theme-color
 
Here is the Theia doc: https://github.com/eclipse-theia/theia/pull/6475/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed